CONTRIBUTING.md

```diff
@@ -179,7 +179,36 @@ virtualenv venv
 # Install openhtf into the virtualenv in dev mode.
 python setup.py develop
 ```
+### MacOS
 
+Install Xcode and Homebrew Mac. Once that is done: 
```

Comment:
Ah, I didn't realize xcode was where we were getting pip from, too. I think it's reasonable to recommend and link to both, as long as we provide a few words explaining why.